docs/index.md

```diff
@@ -1,71 +1,126 @@
+
+If democratic institutions and bodies adopted an open data standard for publishing their registers of interests, tools such as [declared.info](http://declared.info/) could be built cheaply and have far-reaching scope. The public interest would be far better served.
+
+## Status of the data model
+
+We present here the beginnings of an open data standard: a draft data model to represent information declared by people about their interests. The structure of the data (the data model) is informed by:
+
+* the experience of dealing with unstructured, variously formatted information in this domain,
+* an understanding of how tightly the format of declarations can be tied to regulatory requirements and language,
+* a recognition that an 'interest' may cover an extremely wide range of activities, roles and engagements,
+* an assumption that the added value of a structured format is in enabling patterns across declarations to be more easily identified,
+* experience elsewhere in the transparency field which highlights the importance of historical data on declarations.
+
+The data model is a first draft: it's an opener in a conversation. Publishing its details here allows the underlying ideas to feed into further work in this domain. We hope that it might provoke interest and criticism.
+
+The [draft data model](https://docs.google.com/spreadsheets/d/1QCVkxi1B-i3xx1lVJXe1ihhQSrgtOEeAxj8CHoFgTVA/edit#gid=0) is documented in a spreadsheet for convenience. Ultimately a related data standard could be rendered in JSON schema.
+
+## Overview of the data model
 
-## Examples
+![Diagram of data model](_assets/UML-interests-data-model.png)
 
-Table without horizonal scrollbar:
+The data model encapsulates an Interest object, placing it within a Declaration. This allows processes of declaration and re-declaration (or confirmation) to be treated and understood separately from the life-cycle and nature of the interest itself. (For example, a person may be required to declare an interest which actually sits with a family member or business partner. And they may confirm that the same interest exists in a future declaration.)
+
+The Interest object itself is only semi-structured. The model is built on the assumption that it is primarily **dates, people and organisations** associated with interests that people wish to link to other information. (We can imagine that there might be other notable information types - for instance, location. Beginning with the most obvious and widely applicable information types seemed wise.) We leave it up to data publishers to provide in their datasets the precise field names for dates, people and organisations. More information about this feature of the model can be found below under Key features > Flexibility.
+    
+[View the draft data format](https://docs.google.com/spreadsheets/d/1QCVkxi1B-i3xx1lVJXe1ihhQSrgtOEeAxj8CHoFgTVA/edit#gid=0)
```

Comment:
This link appears only a few lines up....

Reply:
Have replaced this with an embedded link under the diagram.